operator/config/crd/bases/operator.kyma-project.io_kymas.yaml

```diff
@@ -123,31 +123,68 @@ spec:
                 description: List of status conditions to indicate the status of a
                   ServiceInstance.
                 items:
-                  description: KymaCondition describes condition information for Kyma.
+                  description: "Condition contains details for one aspect of the current
```

Comment:
this looks broken, is this correct?

Reply:
this is automatically generated, I think it's a controller-gen problem, it reads the description from `metav1.Condition`, and don't know why quote the string, but it's not broken, still valid string.